### DISCUSSION TITLE
Editor templates doesn't works

### DISCUSSION THREAD

**Discussion by rigczTheSky** [dbeaver/dbeaver#34253] [Q&A] [answered]
Hi. In the menu SQL Editor -> Templates i have "sf" defined as a shortcut to "select * from", but when i tap in the editor window: sf[ctrl + space] i've got a message "no proposals".

**Comment by TB-HunBoka** [comment, marked as answer]
Because you are looking for ctrl+alt+space.
Or right click in the sql editor and select SQL Template.
